sdcm/sct_events/base.py

```diff
@@ -156,6 +157,14 @@ def get_registry_filter(self) -> ContinuousRegistryFilter:
 
         return registry_filter
 
+    def _find_event_by_id(self, event_id: Union[uuid.UUID, str]) -> List[Optional[ContinuousEvent]]:
```

Comment:
Are you sure it can return None in the list ? 
Don't see any evidence of that

Reply:
Nope. You're right, this can be an empty list, but not a list of `None`.
Fixed (removed the type hint)